src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -961,38 +1008,44 @@ private static bool HasExplicitNullableConversion(TypeSymbol source, TypeSymbol
 
             if (!source.IsNullableType() && !destination.IsNullableType())
             {
-                return false;
+                return Conversion.NoConversion;
             }
 
-            TypeSymbol sourceUnderlying = source.StrippedType();
-            TypeSymbol destinationUnderlying = destination.StrippedType();
+            TypeSymbol unwrappedSource = source.StrippedType();
+            TypeSymbol unwrappedDestination = destination.StrippedType();
 
-            if (HasIdentityConversion(sourceUnderlying, destinationUnderlying))
+            if (HasIdentityConversion(unwrappedSource, unwrappedDestination))
             {
-                return true;
+                return Conversion.ExplicitNullable;
```

Comment:
It should report ExplicitNullable. Both implicit and explicit conversions exist in this case and you get one or another depending on what you are classifying.
The caller used to report ExplicitNullable in all preexisting cases here even before this change.